solutions/usaco-989.mdx

```diff
@@ -51,4 +51,51 @@ class Main {
 
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	freopen("race.in", "r", stdin);
+	freopen("race.out", "w", stdout);
+	
+	int k, n, x, ans, res, i;
```

Comment:
for the love of god please don't put all your variables in the same place

Reply:
I'll change it.